docs/docs/checklists/analytics.md

```diff
@@ -0,0 +1,36 @@
+## Links `<a>`
+
+- [ ] Has a `data-cta-text` OR `data-link-text`
+    - [ ] If it has class `mzp-c-button` it is a CTA
+    - [ ] If it has class `mzp-c-cta-link` it is a CTA
+    - [ ] If there are two of the same `data-cta-text` include `data-cta-position`
+    - [ ] Does not have both `data-cta-text` and `data-link-text`
+- [ ] If linking to another Mozilla property include `utm` params
```

Comment:
Add @maureenlholland's work on standardizing params in springfield to the docs and reference that here.